src/index.css

```diff
@@ -0,0 +1,131 @@
+:root {
+  --white: #fff;
+  --red: #e50914;
+  --grey: #b3b3b3;
+  --dark-grey: #444;
+  --light-grey: #555;
+  --shadow-black: rgba(0, 0, 0, 0.85);
+  --orange: #e87c03;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-image: url(https://assets.nflxext.com/ffe/siteui/vlv3/02dd3f21-2c34-4989-a48b-827dd9edc446/6fd9c935-5097-4045-9e29-4e1028a6c1fb/PL-pl-20190422-popsignuptwoweeks-perspective_alpha_website_large.jpg);
+
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
+
+
+.login-container__form {
+  background-color: var(--shadow-black);
+  max-width: 15%;
+  margin: 5% auto;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem 5rem
+}
+
+.login-container-form__title {
+  color: var(--white);
+  margin-bottom: 1rem;
+  white-space: nowrap;
+}
+
+.login-container-form__input {
+  padding: 1rem;
+  margin-bottom: .7rem;
+  border-radius: 5px;
+  color: var(--white);
+  background-color: var(--dark-grey);
+  border: 1px solid var(--dark-grey);
+}
+
+.login-container-form__input::placeholder {
+  color: var(--white);
+  font-size: 14px;
+}
+
+.login-container-form__input:focus {
+  color: var(--white);
+  background: var(--light-grey);
+}
+
+.login-container-form__input:invalid:focus {
+  border-bottom: 2px solid var(--orange)
+}
+
+.login-container-form__label {
+  color: var(--white);
+  font-size: 20px;
+  margin-bottom: .5rem
+}
+
+.login-container-form__button {
+  margin-top: 3rem;
+  padding: 1rem;
+  background: var(--red);
+  white-space: nowrap;
+  border-radius: 5px;
+  font-weight: bold;
+  border: 1px solid var(--red);
+  font-size: 14px;
+  color: var(--white);
+  text-align: center;
+}
+
+.login-container-form__button:hover {
+  cursor: pointer;
+}
+
+.login-container__no-account {
+  margin-top: 10rem;
+  display: flex;
+  flex-direction: column;
+  color: var(--grey);
+}
+
+.login-container__registration-link {
```

Comment:
There's no space between "Nie masz konta w serwisie Series Advisor?" and "Zarejestruj się"